packages/hardhat-plugin/src/index.ts

```diff
@@ -311,6 +338,90 @@ ignitionScope
     }
   );
 
+ignitionScope
+  .task("verify")
+  .addPositionalParam("deploymentId", "The id of the deployment to verify")
+  .setDescription(
+    "Verify contracts from a deployment against the configured block explorers"
+  )
+  .setAction(async ({ deploymentId }: { deploymentId: string }, hre) => {
+    const { verify } = await import("@nomicfoundation/ignition-core");
+    const { Etherscan } = await import(
+      "@nomicfoundation/hardhat-verify/etherscan"
+    );
+    const { getApiKeyAndUrls } = await import("./utils/getApiKeyAndUrls");
+    const { shouldBeHardhatPluginError } = await import(
+      "./utils/shouldBeHardhatPluginError"
+    );
+    const { verifyEtherscanContract } = await import(
+      "./utils/verifyEtherscanContract"
```

Comment:
We should be ok with importing this top-level, if they aren't importing anything slow in their top-level.